README.md

```diff
@@ -72,16 +80,28 @@ You can create a file watcher to automatically format your files on save.
 In Android Studio / IntelliJ:
 
 > Install the Dart plugin (see Editor setup) to get automatic formatting of code in Android Studio and IntelliJ.
-To automatically format your code in the current source code window, use Cmd+Alt+L (on Mac) or
-Ctrl+Alt+L (on Windows and Linux). Android Studio and IntelliJ also provides a check box named Format code
-on save on the Flutter page in Preferences (on Mac) or Settings (on Windows and Linux) which will format the
-current file automatically when you save it.
+> To automatically format your code in the current source code window, use Cmd+Alt+L (on Mac) or
+> Ctrl+Alt+L (on Windows and Linux). Android Studio and IntelliJ also provides a check box named Format code
+> on save on the Flutter page in Preferences (on Mac) or Settings (on Windows and Linux) which will format the
+> current file automatically when you save it.
 
 In Visual Studio Code:
 
 > Install the Flutter extension (see [Editor setup](https://flutter.dev/docs/get-started/editor)) to get automatic formatting of code in VS Code.
 >
 > To automatically format the code in the current source code window, right-click in the code window and
-select Format Document. You can add a keyboard shortcut to this VS Code Preferences.
+> select Format Document. You can add a keyboard shortcut to this VS Code Preferences.
 >
 > To automatically format code whenever you save a file, set the _editor.formatOnSave_ setting to true.
+
+## Run code metrics locally
+
+Run the following command to get information about the code metrics
```

Comment:
Maybe add that the following command runs the metrics only in the `lib` folder, use `flutter pub run dart_code_metrics:metrics lib test` to run it in the `test` folder as well.